spec/models/resource_action_spec.rb

```diff
@@ -109,13 +109,17 @@
   end
 
   context "#automate_queue_hash" do
-    let(:button) { FactoryBot.create(:custom_button, :applies_to_class => "Vm") }
+    let(:button) { FactoryBot.create(:custom_button, :applies_to_class => "Vm", :options => {:open_url => true}) }
```

Comment:
`, :options => {:open_url => true}`

do we need this change?

Reply:
No.